src/components/desktop/Header.js

```diff
@@ -1,14 +1,22 @@
+import { CaretDown } from '@styled-icons/boxicons-regular'
 import { User } from '@styled-icons/boxicons-solid'
 import { useTranslation } from 'react-i18next'
+import { NavLink, useLocation } from 'react-router-dom'
 import styled from 'styled-components/macro'
 
+const OnAbout = () => {
+  const location = useLocation()
+  return location.pathname.includes('/about/')
```

Comment:
Check out this link: https://falling-fruit-mj5g6rpcp-hack4impact1.vercel.app/map/about/

What's happening on the header here? How could we change this line to avoid this issue?

Reply:
https://reactrouter.com/web/api/Hooks/useroutematch